tests/unit/domain_handlers/customers.py

```
@@ -151,8 +151,90 @@ def handler(sql_text, params):
+                rows = [["lead", 0]]
+            return MockResult(rows)
+
+        # SELECT FROM customers WHERE tenant_id = :tenant_id AND (name ILIKE OR email ILIKE) — search
+        if (
+            "select" in sql_text
+            and "from customers" in sql_text
+            and ("ilike" in sql_text or "like" in sql_text)
+            and "where tenant_id" in sql_text
+            and "group by" not in sql_text
+            and "order by" not in sql_text
+            and "limit" not in sql_text
```

Pylint (4.0.5)

[refactor] Too many boolean expressions in if statement (8/5)

(R0916)